### DISCUSSION TITLE
Method to access postcode_localities from google result

### DISCUSSION THREAD

**Discussion by s21385** [alexreisner/geocoder#1505] [Q&A] [answered]
Hello, 

I use the geocoder gem with the google API.  In my app a user provides a postcode (not a full address) input.  When a postcode covers more than one locality, google adds the optional field postcode_localities.  Is there a way to access that data?  I looked at the methods for the google class but couldn't find any.  Do you plan on adding a method?

Thanks

**Comment by alexreisner** [comment, marked as answer]
You can call the `#data` method on any `Geocoder::Result` object to access the full data structure returned by the API. Let me know if you can't find what you're looking for in there.

**Reply by s21385** [reply]
Thank you.  Much appreciated.
